### PR TITLE
Adding the simplest Högbom's clean algorithm with a basic test

### PR DESCRIPTION
**Pull request by pgabor**

### CODE REVIEW

File: xrayvision/tests/test_clean.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class TestClean(object):
+    # Typical map sizes even, odd
+    @pytest.mark.parametrize("", [()])
+    def test_hogbom_intensity_change(self):
+        # Create a visibility bag
+        u = [5, 7, 15, 37, 45, 57, 60]
+        v = [1, 3, 7, 27, 37, 1, 57]
+        uv_in = np.array([u, v]).reshape(2, len(u))
+        vis_in = [5+7j, 1+2j, 10-5j, -6+1j, 7j, -7-7j, 5j]
```
Comment (samaloney) [inline xrayvision/tests/test_clean.py]: You could use real vis abilities using by using the map_dft and fake data like in the tests for that method

File: xrayvision/Clean.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        --------
+
+        Notes
+        -----
+
+        """
+        max_intesity = 0.0
+        pos = 0
+        summarized = 0.0
+        for i, j in enumerate(np.absolute(self.vis.vis)):
```
Comment (samaloney) [inline xrayvision/Clean.py]: The standard Hogboom clean is performed in image space
1. Perform a back projection or map_idft on the visibilities to obtain the dirty map (DM)
1. Perform a back projection using on the u, v coordinates from the visibilities (with obsvis set to 1) to obtain the dirty beam (DB)
1. Find the brightest point in the in the DM add to a point source map (position and intensity)
1. Subtract at the position of the bright point the DB * intensity of brightest point * gain
1. Go to 3 if more source or iteration threshold reached
1. Convolve the point source map with an idealised BEAM (small gaussian)
1. Add the residuals from the DM to the step above an return

**Comment by pgabor** [comment]
Something is not 100% with the proposed test or with my new CLEAN class. Some debug is needed.

**Comment by pgabor** [comment]
Now it should be okay. I also fixed up the pep8 issues regarding you code :) Let me know if you have any more suggestions/comments

**Pull request merged**